README.md

```diff
@@ -51,12 +129,31 @@ FRIEND configuration file has following structure:
 Please note that you have to put your own \<path\> to pdf file to be able to use external documentation in a browser.
 
 ## Hint Editor
+
 To make contribution to this project easier, there is a simple config editor.
-![](./Resources/screenshots/HintEditor.png)
```

Comment:
I would prefer to have it like that:
```
To make contribution to this project easier, there is a simple config editor. 
![](./Resources/screenshots/HintEditor.png)

__NOTE:__ It can only be built on MacOS with Xcode 8 or higher. Other systems are not supported.
 
Use ``cmake`` to generate Xcode project.
```